pkg/services/rpcsrv/server.go

```diff
@@ -518,6 +531,51 @@ func (s *Server) handleRequest(req *params.Request, sub *subscriber) abstractRes
 	return resp
 }
 
+// handleInternal is an experimental interface to handle client requests directly.
+func (s *Server) handleInternal(req *neorpc.Request, sub *subscriber) (*neorpc.Response, error) {
+	var (
+		res    interface{}
+		rpcRes = &neorpc.Response{
+			HeaderAndError: neorpc.HeaderAndError{
+				Header: neorpc.Header{
+					JSONRPC: req.JSONRPC,
+					ID:      json.RawMessage(strconv.FormatUint(req.ID, 10)),
+				},
+			},
+		}
+	)
+	reqParams, err := params.FromAny(req.Params)
+	if err != nil {
+		return nil, err
+	}
+
+	s.log.Debug("processing rpc request",
```

Comment:
Then maybe we can change the log message to something like "processing local rpc request" in order to distinguish simple and local requests?

Reply:
OK.